volunteer.html

```diff
@@ -165,45 +163,50 @@ <h4>Treasurer</h4>
 
         <div>
           <div class="bio-with-pic">
-            <img class="bio-pic" src="assets/images/volunteers/rachel.jpg" alt="Rachel Fox headshot">
-            <h3>Rachel Fox</h3>
+            <img class="bio-pic" src="assets/images/volunteers/grace.jpg" alt="Grace Erixon">
+            <h3>Grace Erixon</h3>
             <h4>Board Member</h4>
             <div class="badge-container">
+              <div class="gwc-badge instructor">{I}</div>
+              <div class="gwc-badge ta">[TA]</div>
               <div class="gwc-badge organizer">&lt;O&gt;</div>
             </div>
           </div>
         </div>
 
         <div>
           <div class="bio-with-pic">
-            <img class="bio-pic" src="assets/images/volunteers/Erisheadshot.jpg" alt="Eris Koleszar headshot">
-            <h3>Eris Koleszar</h3>
+            <img class="bio-pic" src="assets/images/volunteers/hauwa.jpg">
+            <h3>Hauwa Muibi</h3>
             <h4>Board Member</h4>
             <div class="badge-container">
-              <div class="gwc-badge instructor">{I}</div>
               <div class="gwc-badge ta">[TA]</div>
+              <div class="gwc-badge speaker">"S"</div>
               <div class="gwc-badge organizer">&lt;O&gt;</div>
             </div>
           </div>
         </div>
 
         <div>
           <div class="bio-with-pic">
-            <img class="bio-pic" src="assets/images/volunteers/laurel.jpg" alt="Laurel Oetken headshot">
-            <h3>Laurel Oetken</h3>
+            <img class="bio-pic" src="assets/images/volunteers/sarahs.jpg" alt="Sarah Sand">
+            <h3>Sarah Sand</h3>
             <h4>Board Member</h4>
             <div class="badge-container">
+              <div class="gwc-badge instructor">{I}</div>
               <div class="gwc-badge organizer">&lt;O&gt;</div>
             </div>
           </div>
         </div>
 
         <div>
           <div class="bio-with-pic">
-            <img class="bio-pic" src="assets/images/volunteers/morgan.jpg" alt="Morgan Smith headshot">
-            <h3>Morgan Smith</h3>
+            <img class="bio-pic" src="assets/images/volunteers/biancaz.jpg">
+            <h3>Bianca Zongrone</h3>
```

Comment:
I failed to mention this, but Bianca got married, and her married name is Bianca Zongrone Jefferson.  While you're here, could you make this change?

Sorry for not mentioning it earlier.